CMakeLists.txt

```diff
@@ -17,36 +14,60 @@ include(CMakeDependentOption)
 option(BUILD_TESTING "Option for enabling testing" ON)
 option(VCPKG_ALLOW_APPLE_CLANG "Option for allowing apple clang, even versions that we don't know will work" OFF)
 option(VCPKG_DEVELOPMENT_WARNINGS "Option for turning on all warnings" ON)
-option(VCPKG_WARNINGS_AS_ERRORS "Set warnings to be errors" ${VCPKG_DEVELOPMENT_WARNINGS})
+option(VCPKG_WARNINGS_AS_ERRORS "Set warnings to be errors" OFF)
+option(VCPKG_BUILD_TLS12_DOWNLOADER "Enable building the tls12-downloader" OFF)
 option(VCPKG_BUILD_FUZZING "Option for enabling vcpkg-fuzz support" OFF)
 option(VCPKG_EMBED_GIT_SHA "Option for to fill in the Git SHA version; off by default to avoid privacy concerns out of official builds" OFF)
 
+
 CMAKE_DEPENDENT_OPTION(VCPKG_BUILD_BENCHMARKING "Option for enabling benchmarking" OFF
     "BUILD_TESTING" OFF)
 
-if(WERROR)
-    message(DEPRECATION "-DWERROR is no longer a supported flag. It doesn't do anything.")
-endif()
 if(DEFINE_DISABLE_METRICS OR VCPKG_DISABLE_METRICS)
     message(DEPRECATION "DEFINE_DISABLE_METRICS / VCPKG_DISABLE_METRICS are now handled by creating a "
         "file vcpkg.disable_metrics next to the binary.")
 endif()
 
+if(VCPKG_BUILD_TLS12_DOWNLOADER AND NOT WIN32)
+    message(FATAL_ERROR "The tls12-downloader binary is only supported on Windows.")
+endif()
+
```

Comment:
```suggestion
```
This seems unnecessarily pedantic; it's off-by-default.

Reply:
sure, wfm.